raster/src/main/scala/geotrellis/raster/io/geotiff/tags/codes/TiffFieldType.scala

```diff
@@ -14,5 +14,7 @@ object TiffFieldType {
   val SignedFractionalsFieldType = 10
   val FloatsFieldType = 11
   val DoublesFieldType = 12
-
+  val LongsFieldType = 16
```

Comment:
I would suggest something like

```scala
val LongsFieldType = 16 // This is 64-bits
```

Just to make sure it is super-obvious that this field is not present in normal TIFFs (that is, "long" means 64-bits in the Java world, whereas others might expect "long long").